shared/chat/conversation/container.js

```diff
@@ -65,53 +78,53 @@ export default connect(
+    onAddParticipant: (participants: Array<string>) => dispatch(newChat(participants)),
+    onAttach: (selectedConversation, filename, title) => dispatch(selectAttachment(selectedConversation, filename, title)),
     onDeleteMessage: (message: Message) => { dispatch(deleteMessage(message)) },
+    onEditMessage: (message: Message) => { dispatch(editMessage(message)) },
+    onLoadAttachment: (selectedConversation, messageID, filename) => dispatch(loadAttachment(selectedConversation, messageID, false, downloadFilePath(filename))),
     onLoadMoreMessages: (conversationIDKey: ConversationIDKey) => dispatch(loadMoreMessages(conversationIDKey, false)),
-    onShowProfile: (username: string) => dispatch(onUserClick(username, '')),
     onOpenFolder: () => dispatch(openFolder()),
+    onOpenInFileUI: (path: string) => dispatch(({payload: {path}, type: 'fs:openInFileUI'}: OpenInFileUI)),
+    onOpenInPopup: (message: AttachmentMessage) => dispatch(navigateAppend([{props: {message}, selected: 'attachment'}])),
     onPostMessage: (selectedConversation, text) => dispatch(postMessage(selectedConversation, new HiddenString(text))),
     onRetryMessage: (outboxID: string) => dispatch(retryMessage(outboxID)),
-    onAddParticipant: (participants: Array<string>) => dispatch(newChat(participants)),
-    onAttach: (selectedConversation, filename, title) => dispatch(selectAttachment(selectedConversation, filename, title)),
-    onLoadAttachment: (selectedConversation, messageID, filename) => dispatch(loadAttachment(selectedConversation, messageID, false, downloadFilePath(filename))),
-    onOpenInPopup: (message: AttachmentMessage) => dispatch(navigateAppend([{selected: 'attachment', props: {message}}])),
-    onOpenInFileUI: (path: string) => dispatch(({type: 'fs:openInFileUI', payload: {path}}: OpenInFileUI)),
+    onShowProfile: (username: string) => dispatch(onUserClick(username, '')),
   }),
   (stateProps, dispatchProps, ownProps: OwnProps) => ({
     ...stateProps,
     ...dispatchProps,
     ...ownProps,
-    onPostMessage: text => dispatchProps.onPostMessage(stateProps.selectedConversation, text),
+    onAddParticipant: () => dispatchProps.onAddParticipant(stateProps.participants.filter(p => !p.you).map(p => p.username).toArray()),
     onAttach: (filename: string, title: string) => dispatchProps.onAttach(stateProps.selectedConversation, filename, title),
-    onLoadMoreMessages: () => dispatchProps.onLoadMoreMessages(stateProps.selectedConversation),
     onLoadAttachment: (messageID, filename) => dispatchProps.onLoadAttachment(stateProps.selectedConversation, messageID, filename),
-    onAddParticipant: () => dispatchProps.onAddParticipant(stateProps.participants.filter(p => !p.you).map(p => p.username).toArray()),
+    onLoadMoreMessages: () => dispatchProps.onLoadMoreMessages(stateProps.selectedConversation),
+    onPostMessage: text => dispatchProps.onPostMessage(stateProps.selectedConversation, text),
   }),
```

Comment:
Let's discuss. if there's a ton of it and its all mixed in and makes the pr harder to understand i know we'd want to pull that out. Cleaning up little pieces as we're revisiting a file thats contained seems like its not harder to discern and making a totally sep PR might create more cognitive overhead